storage/store.go

```diff
@@ -1195,8 +1197,11 @@ func (s *Store) resolveWriteIntentError(wiErr *proto.WriteIntentError, rng *Rang
 		}
 		bArgs.Add(pushArgs)
 	}
+	b := &client.Batch{}
+	b.InternalAddCall(client.Call{Args: bArgs, Reply: bReply})
```

Comment:
Do you see this as the only place where you are going to call this method? Perhaps you can rename it to be more specific.